scripts/README.md

```diff
@@ -0,0 +1,28 @@
+# This directory contains the scripts for the automated deployment of Metal3 as well as virtual bare metal hosts.
+
+## This scripts are specific to fresh Equinix Ubuntu 22.04LTS servers. The scripts will fail with other network or resource configurations.
+- m3.small.x86 is the weakest configuration that will support the Metal3 deploy, but not recommended if deploying virtual bare metal hosts.
+- c2.medium.x86 will support a Metal3 deploy with several virtual bare metal hosts.
+
+### The quickest setup for Metal3 is to deploy an Equinix server with the previously mentioned configurations, then run the following commands below which will automate the entire deployment.
+
+
+1. The following commands deploy Metal3
+```
+cd ~; git clone https://github.com/suse-edge/metal3-demo.git
```

Comment:
We should either refer from the main README to this one (if we're assuming the repo isn't already cloned) or drop this step altogether (if we're assuming we're already reading this from a locally cloned repo).

Reply:
I want to operate under the assumption it's not already cloned, so like you mentioned I'll be adding a reference in the main README. Thank you